### PR TITLE
11 struct refactor create

### PR DESCRIPTION
**Pull request by zacdemi**
Refactored CreateRoute into separate components and css modules.

### CODE REVIEW

File: src/components/common/Card/EntryCard.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <div className="entry-image-holder">
+          <a href={props.entry.sandboxUrl} target="_blank">
+            <img src={props.entry.imageSrc} className="entry-image" />
+          </a>
+        </div>
+        <div className="entry-description">{props.entry.description}</div>
+      </div>
+    </div>
+  );
+}
```
Comment (pyxld-kris) [inline src/components/common/Card/EntryCard.js]: It looks like this file may not be used?

File: src/components/common/Card/EntryCardCollection.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+export default function EntryCardCollection(props) {
+  return (
+    <div className="collection collection--small-cards">
+      {props.data.map((entry, i) => {
+        return <EntryCard entry={entry} key={i} />;
+      })}
+    </div>
+  );
+}
```
Comment (pyxld-kris) [inline src/components/common/Card/EntryCardCollection.js]: It looks like this file may not be used?

File: src/components/common/Card/EntryCard.js (hunk 1 above)
Comment (zacdemi) [reply src/components/common/Card/EntryCard.js]: Yes, duplicate. Forgot to remove this folder.

File: src/components/common/Card/EntryCardCollection.js (hunk 2 above)
Comment (zacdemi) [reply src/components/common/Card/EntryCardCollection.js]: Yes, duplicate. Forgot to remove this folder.

**Pull request merged**